pkg/services/sso/keycloak_service.go

```diff
@@ -495,14 +495,14 @@ func (kc *masService) DeRegisterKasFleetshardOperatorServiceAccount(accessToken
 
 func (kc *masService) RegisterConnectorFleetshardOperatorServiceAccount(accessToken string, agentClusterId string) (*api.ServiceAccount, *errors.ServiceError) { // (agentClusterId string, roleName string) (*api.ServiceAccount, *errors.ServiceError) {
```

Comment:
`RegisterConnectorFleetshardOperatorServiceAccount` and `RegisterKasFleetshardOperatorServiceAccount` are identical